test/storages/vector_storages/test_qdrant.py

```diff
@@ -96,3 +105,135 @@ def test_existing_collection():
         collection_name="test_collection",
     )
     assert storage2.status().vector_count == 2
+
+
+def test_points_crud():
+    tmpdir = tempfile.mkdtemp()
+    collection_name = "test_collection"
+    manager = QdrantManager(path=tmpdir)
+
+    if manager.collection_exists(collection_name):
+        manager.delete_collection(collection_name)
+
+    storage = QdrantStorage(
+        vector_dim=4,
+        path=tmpdir,
+        collection_name=collection_name,
+    )
```

Comment:
What use cases we need to first define `QdrantManager ` and then `QdrantStorage`?